src/gmt_support.c

```diff
@@ -1358,8 +1358,8 @@ GMT_LOCAL bool gmtsupport_is_penstyle (char *word) {
 	if (n == 0) return (false);
 
 	if (!strncmp (word, "dotdash", 7U) || !strncmp (word, "dashdot", 7U) \
-	    || !strncmp (word, "dash", 4U) || !strncmp (word, "dot", 3U) \
-	    || !strncmp (word, "solid", 5U)) return (true);
+		|| !strncmp (word, "dash", 4U) || !strncmp (word, "dot", 3U) \
```

Comment:
Don't replace those spaces by tabs. Using the spaces like that is the only way of keep the alignment with the first character after the parentesis.

Reply:
Sorry, it was an auto think.  I will fix it.